docs/deployment/trimming.md

```diff
@@ -0,0 +1,173 @@
+---
+title: "Trim a .NET MAUI app"
+description: "Learn about the .NET trimmer, which eliminates unused code from a .NET MAUI app to reduce its size."
+ms.date: 10/22/2024
+no-loc: [ ILLink ]
+monikerRange: ">=net-maui-9.0"
+---
+
+# Trim a .NET MAUI app
+
+When it builds your app, .NET Multi-platform App UI (.NET MAUI) can use a linker called *ILLink* to reduce the overall size of the app with a technique known as trimming. ILLink reduces the size by analyzing the intermediate code produced by the compiler. It removes unused methods, properties, fields, events, structs, and classes to produce an app that contains only code and assembly dependencies that are necessary to run the app.
+
+To prevent changes in behavior when trimming apps, .NET provides static analysis of trim compatibility through trim warnings. The trimmer produces trim warnings when it finds code that might not be compatible with trimming. If there are any trim warnings, the app should be thoroughly tested after trimming to ensure that there are no behavior changes. For more information, see [Introduction to trim warnings](/dotnet/core/deploying/trimming/fixing-warnings).
+
+## Trimming behavior
+
```

Comment:
Worth adding a note that MAUI doesn't use `$(PublishTrimmed)`?

Reply:
"Mobile" platforms like iOS, Catalyst, Android, etc. use `$(PublishTrimmed)` by default because they are "self-contained", but a developer would likely never put it in their project file and just rely on defaults.